test/extended/util/annotate/rules.go

```diff
@@ -74,10 +74,19 @@ var (
 			// https://bugzilla.redhat.com/show_bug.cgi?id=2019115
 			`\[sig-storage\].*\[Feature:vsphere\]`,
 			// Also, our CI doesn't support topology, so disable those tests
-			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV \(delayed binding\)\] topology should fail to schedule a pod which has topologies that conflict with AllowedTopologies`,
-			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV \(delayed binding\)\] topology should provision a volume and schedule a pod with AllowedTopologies`,
-			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV \(immediate binding\)\] topology should fail to schedule a pod which has topologies that conflict with AllowedTopologies`,
-			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV \(immediate binding\)\] topology should provision a volume and schedule a pod with AllowedTopologies`,
+			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV (delayed binding)\] topology should fail to schedule a pod which has topologies that conflict with AllowedTopologies`,
```

Comment:
When you update for the verify test, can you also reference the bug or jira card that will allow these to pass so we know when we can remove the skip?